packages/integration-tests/migrations/1580658856631_author.ts

```diff
@@ -158,4 +158,17 @@ export function up(b: MigrationBuilder): void {
     parent_book_review_id: foreignKey("book_reviews", { notNull: false, unique: true }),
     text: "text",
   });
+
+  // testing for a required polymorphic field
```

Comment:
I get this is "a favorite thing can be owned by book/publisher/author"...what's the `author_favorites` for? Turning it into a m2m?

Reply:
I've revised this PR to not add any new entities, per my comment [here](https://github.com/stephenh/joist-ts/pull/291/files#r844309866). I feel like this muddied the water unnecessarily.